docs/guides/src/main/server/configuring-hostname.adoc

```diff
@@ -0,0 +1,177 @@
+<#import "/templates/guide.adoc" as tmpl>
```

Comment:
there are 2 cross-refs ( ` <@links.server id="hostname"/>` ) I've found in the docs to the hostname.adoc - pls change these to use ` <@links.server id="configuring-hostname"/>` instead so they continue to work properly.